.github/workflows/ci.yml

```diff
@@ -123,6 +123,50 @@ jobs:
           name: ${{ steps.pack.outputs.package_file }}
           path: ${{ steps.pack.outputs.package_file }}
 
+  create-workspace-packages:
```

Comment:
Can this run separate from prebuilds-*?
At the moment this job seems to copy the artifacts from prebuild, but do we need them for this job?